### PR TITLE
Update to get in sync with asp.net RC2

### PR DESCRIPTION
**Pull request by brentschmaltz**
dnx451 -> net451 in test.
netstandard1.3, dotnet5.4 -> netstandard1.4, dotnet5.5 in src.
To be in sync with asp.net
Updated versions of System* assemblies.
Removed dependencies: System.Security.Principal, System.Security.Cryptography.OpenSsl
Update Newtonsoft.Json -> 8.0.3

#354 
#350

### CODE REVIEW

**Comment by brentschmaltz** [comment]
@tushargupta51 @PinpointTownes

File: test/Microsoft.IdentityModel.Logging.Tests/project.json [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
-    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*"
+    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*",
+    "xunit": "2.1.0"
   },
   "testRunner": "xunit",
   "frameworks": {
     "netstandardapp1.5": {
-      "imports": [ "dnxcore50", "portable-net451+win8" ]
+      "dependencies": {
+        "dotnet-test-xunit": "1.0.0-dev-*",
```
Comment (tushargupta51) [inline test/Microsoft.IdentityModel.Logging.Tests/project.json]: We don't need to include these dependencies here, they gets pulled in by Microsoft.IdentityModel.Tokens.Tests. I took them off in a recent PR. Was it breaking without this?

File: src/Microsoft.IdentityModel.Tokens/project.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
   "description": "Includes types that provide support for cryptographic operations.",
   "dependencies": {
     "Microsoft.IdentityModel.Logging": "1.0.0-*",
-    "Newtonsoft.Json": "6.0.6"
+    "Newtonsoft.Json": "8.0.3"
```
Comment (PinpointTownes) [inline src/Microsoft.IdentityModel.Tokens/project.json]: nit: IIRC, the ASP.NET team mainly uses JSON.NET 8.0.2. Maybe you should ping them so they eventually migrate to the latest bits before RC2 is released.

File: test/Microsoft.IdentityModel.Logging.Tests/project.json [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+        "System.Diagnostics.Process": "4.1.0-*"
+      },
+      "imports": [
+        "dnxcore50",
+        "portable-net451+win8"
+      ]
     },
-    "dnx451": {
+    "net451": {
+      "frameworkAssemblies": {
```
Comment (PinpointTownes) [inline test/Microsoft.IdentityModel.Logging.Tests/project.json]: nit: you should be able to remove this empty node.

File: test/System.IdentityModel.Tokens.Saml.Tests/project.json [hunk 4]
```diff
@@ -9,9 +10,22 @@
...
+        "System.Diagnostics.Process": "4.1.0-*"
+      },
+      "imports": [
+        "dnxcore50",
+        "portable-net451+win8"
+      ]
     },
-    "dnx451": {
+    "net451": {
+      "frameworkAssemblies": {
```
Comment (PinpointTownes) [inline test/System.IdentityModel.Tokens.Saml.Tests/project.json]: Same remark :smile:

**Comment by PinpointTownes** [comment]
LGTM :shipit:

File: test/Microsoft.IdentityModel.Logging.Tests/project.json (hunk 1 above)
Comment (brentschmaltz) [inline test/Microsoft.IdentityModel.Logging.Tests/project.json]: asp.net is still including them.  I will take them off and see what happens

File: src/Microsoft.IdentityModel.Tokens/project.json (hunk 2 above)
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Tokens/project.json]: aspnet/security is using 8.0.3

File: test/System.IdentityModel.Tokens.Saml.Tests/project.json (hunk 4 above)
Comment (brentschmaltz) [inline test/System.IdentityModel.Tokens.Saml.Tests/project.json]: Same remark :-) :-)

File: test/Microsoft.IdentityModel.Logging.Tests/project.json (hunk 1 above)
Comment (brentschmaltz) [inline test/Microsoft.IdentityModel.Logging.Tests/project.json]: Build fails without them, for me anyway,

File: test/Microsoft.IdentityModel.Logging.Tests/project.json (hunk 3 above)
Comment (brentschmaltz) [inline test/Microsoft.IdentityModel.Logging.Tests/project.json]: thanks, I will see what happens. seems to build fine without them. I will remove them.

**Pull request merged**